facebook_connector.py

```diff
@@ -91,11 +100,14 @@ def get_cleaned_fully_merged_messages(strip_html_content=True,
                     content = message.content
```

Comment:
Whoops, forgot to remove that. It was a failed attempt of trying to remove the warnings